src/demo/demo.ts

```diff
@@ -1,13 +1,19 @@
 import Driftory from '../library/driftory';
 
 document.addEventListener('DOMContentLoaded', () => {
-  const container = document.querySelector('.driftory-viewer-container') as HTMLDivElement;
+  const container = document.querySelector('.driftory-viewer-container') as HTMLDivElement | null;
```

Comment:
Just remove the `as HTMLDivElement | null`

If you are going to check for existence then this isn't really necessary.

Reply:
OK, I tried making this change but `document.querySelector` returns an `Element` type, not a `HTMLElement` type and the stye prop doesn't exist on the `Element` type so we have no choice but to do this.

Reply:
Yup, that's the same process I went through :-) thank you for confirming!

Reply:
We should probably add a comment there to stop people going through the same loop we both just went through 😕

Reply:
Okay, I've added a comment as part of https://github.com/pixfabrik/driftory-viewer/pull/20